### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1371,6 +1371,7 @@ allwys->always
 almostly->almost
 almsot->almost
 alo->also
+aloable->allowable
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:1374]: This seems quite a stretch!

The only hit grep.app had was avaloable which should be to available so would you mind adding that?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -23963,6 +23966,7 @@ self-comparisson->self-comparison
 self-contianed->self-contained
 self-referencial->self-referential
 self-refering->self-referring
+selt->set, self, sold,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:23969]: Suggest seat.

I'm struggling to see sold though?

From looking at grep.app I think it should be self and then set.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -28809,6 +28814,7 @@ vissible->visible
 visted->visited
 visting->visiting
 vistors->visitors
+visualyse->visualize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28817]: Can you suggest visualise first.

Can you also add an equivalent typo for visualiser/visualizer.

**Review by peternewman — CHANGES_REQUESTED** [review]
A few comments

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:1374]: stretch, you say? github says otherwise: https://github.com/search?q=%22aloable%22&type=code

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:1374]: Okay, although they seem to be one typo in lots of forks of the same repo. I guess it's fixed upstream hence why grep.app isn't returning it or something?

Anyway fair enough.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:23969]: `He selt it last week` a foreign speaker could make this mistake.

**Comment by luzpaz** [comment]
Revisions added to 8b02264, please confirm

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:23970]: Okay, I thought that would be more likely to be "selled it" (do you want to add that too if it's not there.

Can you swap self and set though please.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Thanks, can you add the `visualyser` equivalent please.

**Pull request merged**